src/main/java/org/testeditor/fixture/host/Locator.java

```diff
@@ -12,30 +12,154 @@
  *******************************************************************************/
 package org.testeditor.fixture.host;
 
+import java.lang.reflect.Constructor;
+
+/**
+ * The Locator is a representation of a range on a Mainframe host screen.<br>
+ * This could be represented through a declaration of a start and an end point
+ * in two ways.
+ * <ol>
+ * <li><b>start row ; start column ; end row ; end column</b></li>
+ * <li><b>start row ; start column ; width (number of characters)</b></li>
```

Comment:
Apparently there are two different kinds (or types) of locator. Would it make sense to put them into two separate classes that inherit from a common base class?